.travis.yml

```diff
@@ -12,10 +12,10 @@ addons:
       - wkhtmltopdf
 
 env:
-  - VERSION="9.0" LINT_CHECK="1"
-  - VERSION="9.0" ODOO_REPO="odoo/odoo" LINT_CHECK="0"
-  - VERSION="9.0" ODOO_REPO="OCA/OCB" LINT_CHECK="0"
-  - VERSION="9.0" UNIT_TEST="1" LINT_CHECK="0"
+  - VERSION="10.0.0.0.0" LINT_CHECK="1"
+  - VERSION="10.0.0.0.0" ODOO_REPO="odoo/odoo" LINT_CHECK="0"
+  - VERSION="10.0.0.0.0" ODOO_REPO="OCA/OCB" LINT_CHECK="0"
+  - VERSION="10.0.0.0.0" UNIT_TEST="1" LINT_CHECK="0"
 
```

Comment:
The travis file version is different than manifest one.
Keep VERSION="10.0" LINT_CHECK="1"
Not VERSION="10.0.0.0.0"